src/main/java/seedu/mealcompanion/command/recipe/RecipeRandomCommand.java

```diff
@@ -40,9 +40,13 @@ public void execute(MealCompanionSession mealCompanionSession){
             int index = getRandomIndex(recipes);
             Recipe recipe = recipes.getRecipe(index);
             mealCompanionSession.getUi().printMessage("A random recipe is chosen:");
+            if (hasAllergen(recipe, mealCompanionSession.getAllergens())) {
+                mealCompanionSession.getUi().printMessage("IMPORTANT: You are allergic to the ingredient(s) in this " +
+                        "recipe. Proceed with care or run command again to get a different random recipe.");
+            }
             mealCompanionSession.getUi().printMessage(recipe.toString());
         } catch (MealCompanionException e) {
-            mealCompanionSession.getUi().printMessage(String.valueOf(e));
+            mealCompanionSession.getUi().printMessage(e.getMessage());
```

Comment:
Would it make more sense to have a catch-all outside of the execute function to reduce the repetition of this line? Good fix for now, but something we should look into to avoid repetition.

Reply:
A catch-all on a higher level definitely helps to reduce repetition! Lets look into this idea in our next meeting.